codesmell/src/main/resources/application.yml

```
@@ -0,0 +1,2 @@
+server:
+  port: 8081
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)